Pflichtenheft.md

```diff
@@ -247,10 +255,26 @@ Nach der Installation auf dem Kunden-Computer, wird die Software nur auf diesem
  - Preis
  - Status (geplant, beauftragt, in Durchführung, beendet, abgebrochen)
  - Anmerkungen (z.B. Abbruchgrund)
+ - Art (Einzelflug, Flug mit Zwischenaufenthalten, Zeitcharter)
+
+Zusätzlich wird, abhängig von der Art des Auftrags, folgendes gespeichert: 
+ 
+ - Art = Einzelflug oder Flug mit Zwischenaufenthalten
+  - Von (Flughafen/Ort)
+  - Nach (Flughafen/Ort)
+  - Zwischenziele (Flughäfen/Orte)
+  - Anzahl der Passagiere (bei Einzelflug und Flug mit Zwischenaufenthalten)
+ - Art = Zeitcharter
+  - Charterdauer
+ 
+
+## 5.6. Rechnungen
+
+Zu Rechnungen werden die folgenden Informationen erfasst:
 
-## 5.6. Mahnwesen
  - Auftrag (Link zum Auftrag)
- - Status (Rechnung erstellt, verschickt, bezahlt; Erinnerung 1, 2; Mahnung 1,2; Rechnung nicht bezahlt)
+ - Status (erstellt, verschickt, bezahlt, 1. Erinnerung verschickt, 2. Erinnerung verschickt, 1. Mahnung verschickt, 2. Mahnung verschickt, Rechnung nicht bezahlt)
```

Comment:
Der Status "Rechnung nicht bezahlt" kann weggelassen werden. Die Rechnung befindet sich immer in diesem Status, es sei denn, sie wurde bezahlt.